scripts/beck/st/error_injection.py

```diff
@@ -1,66 +1,78 @@
 parser.add_argument('-v', action='store_true', default=False, dest='verify', help='Read full Metadata and Print all the Records entry counts')
 parser.add_argument('-a', action='store_true', default=False, dest='allErr', help='Induce Error in All Record at Random place')
 parser.add_argument('-gmd', action='store_true', default=False, dest='allGMD', help='Induce Error in All GMD type of Record at Random place')
 parser.add_argument('-dmd', action='store_true', default=False, dest='allDMD', help='Induce Error in All DMD type of Record at Random place')
 parser.add_argument('-512k', action='store_true', default=False, dest='err512k', help='Induce 512K bytes error in Metadata')
 parser.add_argument('-huge', action='store_true', default=False, dest='hugeCorruption', help='Induce Huge amount of corruption in Metadata')
+parser.add_argument('-seed', action='store', default=0, type=float, dest='seed', help='Seed for generating Random errors corruption')
 
 args = parser.parse_args();
-#print("Random {0} No of Errors {1} Record Types {2} File name {3} Verify {4}".format(args.random, args.noOfCorr, args.recordType, args.filename, args.verify ))
 
 results = parser.parse_args()
-print('Induce Random Error        = {!r}'.format(args.random))
-print('No of Error induce         = {!r}'.format(args.noOfErr))
-print('Record Type                = {!r}'.format(args.recordT))
-print('Metadata file path         = {!r}'.format(args.file))
-print('Verify Record entries      = {!r}'.format(args.verify))
-print('Induce Error in All Record = {!r}'.format(args.allErr))
-print('Induce Error in GMD Record = {!r}'.format(args.allGMD))
-print('Induce Error in DMD Record = {!r}'.format(args.allDMD))
-print('Induce 512k errors         = {!r}'.format(args.err512k))
-print('Induce huge corruption     = {!r}'.format(args.hugeCorruption))
-
-filename  = args.file
-recordType= args.recordT
+logger.info('Induce Random Error        = {!r}'.format(args.random))
+logger.info('No of Error induce         = {!r}'.format(args.noOfErr))
+logger.info('Record Type                = {!r}'.format(args.Record_Type))
+logger.info('Metadata file path         = {!r}'.format(args.mfile))
+logger.info('Verify Record entries      = {!r}'.format(args.verify))
+logger.info('Induce Error in All Record = {!r}'.format(args.allErr))
+logger.info('Induce Error in GMD Record = {!r}'.format(args.allGMD))
+logger.info('Induce Error in DMD Record = {!r}'.format(args.allDMD))
+logger.info('Induce 512k errors         = {!r}'.format(args.err512k))
+logger.info('Induce huge corruption     = {!r}'.format(args.hugeCorruption))
+logger.info('Seed for random number     = {!r}'.format(args.seed))
+
+filename  = args.mfile
+recordType= args.Record_Type
 noOfCorr  = args.noOfErr
 
+if args.seed != 0:
+	seed = args.seed
+	logger.info("Seed used: {}".format(seed))
+else:
+	seed = time.time()
+	logger.info("Seed used: {}".format(seed))
+
+random.seed(seed)
```

Comment:
From here, I know the 'seed' is only used to initialize the "random" library: to initialize the random generation.

Please explain this in help.